README.md

```diff
@@ -189,13 +189,22 @@ nginx rules to redirect the normal pages to a 503 maintenance page.
 * Default: true
 * Description: A field to enabled/disable network policies.
 
+#### `general.networkPolicies.cilium.enabled`
```

Comment:
how is this different from `general.networkPolicies.enabled` if cni is cilium?
I think this is only disabling the Cilium specific ones? Why would you use cilium as `general.cni` and disable the cilium policies?